service/src/main/java/org/ehrbase/dao/access/jooq/FolderAccess.java

```diff
@@ -496,10 +496,10 @@ public static I_FolderAccess buildPlainFolderAccess(final  I_DomainAccess domain
         }
         //pstmt.setObject(11, jsonObject);
 
-        if (folder.getDetails() != null) {
+        /* if (folder.getDetails() != null) {
```

Comment:
Just taking a look at the snippet I'm not sure what happens here, but formally `details` in a Folder can be empty/null, see: https://specifications.openehr.org/releases/RM/latest/common.html#_folder_class (just as a note)